modules/product/product.resolver.js

```diff
@@ -2,7 +2,14 @@ const productsService = require('./product.service');
 
 const productsQuery = {
   getProductsById: (parent, args) => productsService.getProductsById(args.id),
-  getProducts: (parent, args) => productsService.getProducts(args),
+  getProducts: async (parent, args) => {
+    const products = await productsService.getProducts(args);
```

Comment:
Pagination should be created inside the service and returned from the one method call

Reply:
fixed